external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -368,6 +420,10 @@ pub struct Symbols {
     def_info: DefMap,
     /// IDE Annotation Information from the Compiler
     pub compiler_info: CompilerInfo,
+    /// Cursor information gathered up during analysis
+    pub cursor_context: Option<CursorContext>,
+    /// Typed Program
+    pub typed_ast: Option<T::Program>,
```

Comment:
I was trying to avoid storing the whole typed AST with symbols but if we go that route, I wonder if there is some way to optimize away other things we are storing with symbols (e.g., from `DefInfo` in `DefMap`)

Reply:
That would be a huge win longer-term, I think!